base/db/R/query.trait.data.R

```diff
@@ -17,6 +17,9 @@
 ##' converted from measurement temperature to \eqn{25^oC} via the arrhenius equation.
 ##'
 ##' @param trait is the trait name used in the database, stored in variables.name
+##' @param con Database connection object
+##' @param update.check.only if TRUE, returns results but does not print summaries
+##' @param ... further parameters passed to the fitting function, in the case of loess, for example, 'span = .9', or 'family = 'symmetric''
```

Comment:
It doesn't look like `...` is actually passed to anything.  Did you mean to add `...` somewhere in the function body?  If not, it might be best to document this as "unused currently"